### ISSUE TITLE
Disabled INFO and DEBUG logging after importing autosklearn.regression

### ISSUE THREAD

**Issue by m-nez** [automl/auto-sklearn#482]
Merely importing autosklearn.regression disables DEBUG and INFO level logging.
```
import logging
import autosklearn.regression
logging.basicConfig(level=logging.DEBUG)
logging.debug("TEST")
```
Should output to stderr:
```
DEBUG:root:TEST
```
Instead it doesn't output anything.
On the other hand if "import autosklearn.regression" is removed the output is correct.

auto-sklearn 0.3.0
python 3.6.5

**Comment by yMorakkam** [comment]
I had the same issue after calling `fit_ensemble` method. It's due to the fact that  ensemble_builder initializer create a new logger with the method `autosklearn.util.logging_.get_logger`.

**Comment by mfeurer** [comment]
This should be fixed by #505. Please reopen if this problem still exists.
